hpack/table.py

```diff
@@ -105,6 +111,13 @@ class HeaderTable(object):
 
     STATIC_TABLE_LENGTH = len(STATIC_TABLE)
 
+    STATIC_TABLE_MAPPING = {}
+    for index, (name, value) in enumerate(STATIC_TABLE):
+        header_name_search_result = STATIC_TABLE_MAPPING.setdefault(name, _HeaderNameSearchResult())
+        header_name_search_result.positions[value] = index
+        min_index = header_name_search_result.min
+        header_name_search_result.min = min(min_index, index) if min_index is not None else index
```

Comment:
Don't use a `staticmethod`: there's no need. Let's just write an actual function: not all code needs to be attached to classes.